lnrpc/rpc.proto

```diff
@@ -1327,6 +1327,9 @@ message ChannelBalanceResponse {
 
     /// Sum of channels pending balances denominated in satoshis
     int64 pending_open_balance = 2 [json_name = "pending_open_balance"];
+
+    /// Sum of channels remote balances denominated in satoshis
+    int64 remote_balance = 3 [json_name = "remote_balance"];
```

Comment:
Hey, thanks for the review. I agree `_sat ` suffix makes sense, but I noticed that of all satoshi denominated fields in rpc.proto, only 3 have suffix `_sat`. Looks like satoshis is the default and `_msat` is used when needed. Isn't that the case? Also as `ChannelBalanceResponse` has only such fields, it will look oddly inconsistent if only one of them is _sat'ed. 

What to do next?
1) Rename `remote_balance` to `remote_balance_sat`.
2) Add suffix `_sat` to all fields in `ChannelBalanceResponse`.
3) Keep it as it is.
4) Close this PR and refactor rpc.proto at later stage (v2 ?).

I was also thinking about renaming `balance` to `local_balance`, as now there is `remote_balance`. What do you think?

Thanks!